.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+language: python
+
+services:
+  - docker
+
+env:
+  global:
+    - PORTS="8069:8069"
+    - LINKS="db:db"
+    - DAEMONIZE="1"
+    - ODOO_VERSION="10.0"
+
+  matrix:
+    - TESTS="1" ODOO_VERSION="10.0" DOCKER_CONTEXT="./tests/odoo10"
+    - TESTS="1" ODOO_VERSION="9.0" DOCKER_CONTEXT="./tests/odoo9"
+    - LINT_CHECK="1"
+
+before_install:
+  - docker build -t "tecnativa/odoo-base:${ODOO_VERSION}" --build-arg "ODOO_VERSION=${ODOO_VERSION}" ./
+  - docker pull kiasaki/alpine-postgres
```

Comment:
This is not the postgres version we support, it should be the official one `postgres:9.6-alpine`